h2o-admissibleml/src/main/java/hex/Infogram/InfogramExtension.java

```diff
@@ -0,0 +1,18 @@
+package hex.Infogram;
+
+import hex.tree.xgboost.XGBoostExtension;
+import org.apache.log4j.Logger;
+import water.AbstractH2OExtension;
+
+public class InfogramExtension extends AbstractH2OExtension {
```

Comment:
this class is not actually needed for anything - please remove

Reply:
@michalk:  Attempt to delete and got the following warning:
<img width="967" alt="image" src="https://user-images.githubusercontent.com/7231712/145489142-0bdf82b3-a9ae-4a4a-bba0-802dfd02056d.png">

I copied this from XGBoost stuff

Reply:
@wendycwong it is not actually needed - please remove the service file, and all usages - in your case it is not needed - it doesn't do anything

Reply:
Done removing